README.md

```diff
@@ -34,73 +34,94 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
-2. Name two big differences between ```display: block;``` and ```display: inline;```.
+   Semantic HTML is an markup concept where you use specific tags such as `section`, `header`, or `nav` to specifically point out to entities reading the code (people, search engine, screen readers, etc.) what the markup was intended for. This is not to be confused with Presentational HTML where tags are used to separate code into containers for presentation purposes and styling later.
+
+2. Name two big differences between `display: block;` and `display: inline;`.
+
+`display: block;`
+
+    a. Takes up the entire width of its container, so it will be on a new line/row than its predecessors, and all content after it will also be on another line.
+
+    b. Respects all margins and all padding
+
+`display: inline;`
+
+    a. Allows elements to sit to their left and right
+    b. Only respects left and right margins + padding
 
 3. What are the 4 areas of the box model?
 
-4. While using flexbox, what axis does the following property work on: ```align-items: center```?
+   1. Margin
+   2. Border
+   3. Padding
+   4. Content
+
+4. While using flexbox, what axis does the following property work on: `align-items: center`?
+   Cross axis.
 
 5. Explain why git is valuable to a team of developers.
```

Comment:
Great detailed answer. Version control is one of the most important tools you'll use as a developer and it's important to understand why it's so necessary.